src/app/constants/time.constant.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Milliseconds in an hour
+ */
+export const oneHour = 60 * 60 * 1000;
```

Comment:
Should add `'Millis'` as a postfix to both variable names. Makes it more obvious what value the variable holds